src/API.ts

```diff
@@ -82,23 +94,37 @@ const skypeTokenAuth = async (chatToken: OmnichannelChatToken): Promise<Response
     }
 }
 
+
+const getTypeFromFile = (type: string, name: string, operation: string) => {

```

Comment:
Would `createFileResource` be a better naming? It technically creates the API resource based on certain condition then append it in the URL to create the API endpoint.

Reply:
I dont think is creating anything here, is just evaluating the type of file, to return a value based on the arguments, is  plain classification .

Reply:
I don't see it as creation, is just defining the value for an attribute to be used for others based on the input arguments, is plain classification,

Reply:
maybe defineType... Ill think on a better name...